lib/alex_koin/slack.ex

```diff
@@ -3,19 +3,29 @@ defmodule AlexKoin.SlackRtm do
     :alex_koin, :slack_module, Slack.Sends
   )
 
+  @koin_bot_id Application.get_env(:alex_koin, :koin_bot_id)
+
   alias AlexKoin.SlackCommands
 
   def handle_connect(slack, state) do
     IO.puts "Connected as #{slack.me.name}"
     {:ok, state}
   end
 
-  def handle_event(message = %{type: "message", text: "<@UC37P4L3Y> " <> text, user: user}, slack, state) do
-    SlackCommands.get_or_create(user)
-    |> create_reply(message, message_type(text)) # returns tuple {text, message_ts}
-    |> send_raw_message(message.channel, slack)
+  def handle_event(message = %{type: "message", channel: "D" <> _rest, user: user}, slack, state) do
+    # Ignore if it's a message in a DM from the bot itself
+    if user == @koin_bot_id do
+      {:ok, state}
+    else
+      # We care because we don't have to thread in a DM.
+      handle_msg(user, message, message_type(message.text), slack, state)
+    end
+  end
 
-    {:ok, state}
+  def handle_event(message = %{type: "message", text: "<@" <> @koin_bot_id <> "> " <> text, user: user}, slack, state) do
+    IO.puts inspect(message)
```

Comment:
There's also `IO.inspect` that you could use here, but I really don't like the idea of outputting to the console all over the place. If we need to log stuff, we might want to consider just using the logs.